Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -155,14 +152,14 @@ static void launchWithExtensionKit(ProcessLauncher& processLauncher, ProcessLaun
     ASSERT(!m_xpcConnection);
 
 #if USE(EXTENSIONKIT)
-    auto handler = [](ThreadSafeWeakPtr<ProcessLauncher> weakProcessLauncher, _SEExtensionProcess* process, ASCIILiteral name, NSError* error)
+    auto handler = [](ThreadSafeWeakPtr<ProcessLauncher> weakProcessLauncher, ExtensionProcess process, ASCIILiteral name, NSError* error)
     {
         if (!weakProcessLauncher.get()) {
-            [process invalidate];
+            process.invalidate();
             return;
         }
         if (error) {
-            NSLog(@"Error launching process %@ error %@", process, error);
+            NSLog(@"Error launching process: %@", error);
```

Comment:
Do we really need this in addition to the RELEASE_LOG_FAULT ?

Reply:
That is a good point, I have removed this in the latest patch, along with addressing the other issues.

Thanks for reviewing!